cg/meta/upload/gisaid/gisaid.py

```diff
@@ -276,9 +279,16 @@ def get_accession_numbers(self, case_id: str) -> Dict[str, str]:
             with open(str(log_file.absolute())) as log_file:
                 log_data = json.load(log_file)
                 for log in log_data:
-                    if log.get("code") != "epi_isl_id":
+                    if log.get("code") == "epi_isl_id":
+                        log_message = log.get("msg")
+                    elif log.get("code") == "validation_error" and "existing_ids" in log.get("msg"):
+                        log_message = (
+                            f'{log.get("msg").split(";")[0]}; '
```

Comment:
Is this correct with splitting on ; then looking at [0]? I guess it is correct if it works, but I do not understand this part

Reply:
yes it works - also thats what we do for successful uploads aswell